modin/pandas/test/test_io.py

```diff
@@ -1036,6 +1036,40 @@ def test_from_csv(make_csv_file, nrows):
     df_equals(modin_df, pandas_df)
 
 
+def dummy_decorator():
+    """A problematic decorator that does not use `functools.wraps`. this introduce unwanted local variables for
+    inspect.currentframe.
+    """
+
+    def wrapper(method):
+        def wrapped_function(self, *args, **kwargs):
+            result = method(self, *args, **kwargs)
+            return result
+
+        return wrapped_function
+
+    return wrapper
+
+
+@pytest.mark.parametrize("nrows", [123, None])
+def test_from_csv_within_decorator_parse_args_correctly(make_csv_file, nrows):
```

Comment:
Can you try to make test names shorter?

Reply:
renamed to test_from_csv_within_decorator